app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,49 @@
+package com.codesoom.assignment.controllers;
+
+// TODO
+// 1. GET /tasks (done)
+// 2. GET /tasks/{id} (done)
+//  -> 없을 경우 not found 처리
+// 3. POST /tasks (done)
+// 4. PUT/PATCH /tasks/{id} (done)
+//  -> PUT은 어떻게 하지? PUT과 PATCH mapping을 같이 쓸 순 없나?
+// 5. DELETE /tasks/{id} (done)
+// 6. taskService를 만들기 (done)
+
+import com.codesoom.assignment.models.Task;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import com.codesoom.assignment.services.TaskService;
+
+@RestController
+@RequestMapping("/tasks")
+@CrossOrigin
+public class TaskController {
+    TaskService taskService = new TaskService();
```

Comment:
이렇게 선언한 `TaskService`는 `TaskController` 클래스 바깥에서 참조할 수 없도록 접근 제한을 최소로 지정해 주면 어떨까요?
Java로 코딩할 때에는 가능한 한 많은 멤버를 `private`으로 선언해서 외부의 접근을 최대한 차단하고, 공개된 메소드를 통해서만 상태 변화를 허용하도록 작업해 보세요.

```suggestion
    private TaskService taskService = new TaskService();
```